benchmarks/kubectl-mtb/hack/ci-test.sh

```diff
@@ -17,5 +17,12 @@ export PATH=$(go env GOPATH)/bin:$PATH
 mkdir -p $(go env GOPATH)/bin
 hack_dir=$(dirname ${BASH_SOURCE})
 
-echo "Running 'make test'"
-make test -C "$hack_dir/.."
\ No newline at end of file
+# install and setup kind according to https://kind.sigs.k8s.io/docs/user/quick-start/
+echo "Setting up Kind"
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-linux-amd64
```

Comment:
is it worth checking if kind is already installed in the path before attempting an install?

Reply:
@rjbez17 The tests would run on prow which doesn't have kind cluster installed by default.

Reply:
Yeah, but it would be nice as a developer to be able to run the same script to debug ci issues? Maybe not though since `make test` is all you need

Reply:
@rjbez17 I have changed it to `make unit-tests` which will first check if kind is installed or not and will delete the cluster after running the tests.